CHANGELOG.md

```diff
@@ -1,18 +1,45 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- Menu to components
+- Menu story to storybook
+
+### Changed
+
+- README index with Menu and default values for Dialog
+- README index formatting
+- Dialog story event names
+- let prettier redo some formatting
+
```

Comment:
Does this get moved to a new release section here in the changelog automatically by semantic-release?

Reply:
I semantic release seems to only be writing notes in the github release, so I should do this manually or else figure out how to configure it to write the notes here.